torch/_dynamo/variables/functions.py

```diff
@@ -689,9 +689,14 @@ def wraps(fn):
             return variables.LambdaVariable(wraps)
         else:
             try:
+                print("here")
```

Comment:
remove debug prints

Reply:
Forgot about them, thanks for point out @mlazos !

Reply:
Gentle ping @mlazos  , made the requested changes :blush: